wgengine/userspace.go

```diff
@@ -381,6 +383,7 @@ func (e *userspaceEngine) Reconfig(cfg *wgcfg.Config, routerCfg *router.Config)
 	}
 
 	e.logf("wgengine: Reconfig done")
+	e.logf("wgengine: new peer list: %s", strings.Join(peerKeys, " "))
```

Comment:
Usually this is the right thing to do, but in the case of this particular log message, because of how we want to use it, I think we want to do something slightly different. As written, this message will print out only when wireguard is reconfigured, so:
1. This might be too rare, if the network reconfigures. Processing eg. 1 hour or 1 day of logs is not guaranteed to include the peer list.
2. This might be too frequent. We'll reprint the peer list every time wgengine reconfigures, even if it hasn't changed.

What we want instead is:
1. It must print every time the peer list changes, right away, before any more Peers: lines get printed.
2. It must print at least every n seconds, so that we only have to analyze a relatively short chunk of logs and we'll be guaranteed that the message has printed at least once.

Thus, I'd suggest printing out the peer list right before the existing Peers: line. You'd want something like "if (not_printed_in_last_5_minutes) or (new_peer_list != old_peer_list) then (print new_peer_list)".